src/components/NavBar.js

```diff
@@ -0,0 +1,29 @@
+import { FaArrowAltCircleLeft, FaMicrophone } from 'react-icons/fa';
+import { SlSettings } from 'react-icons/sl';
+import { Link } from 'react-router-dom';
+import MealList from '../containers/MealList';
+import '../styles/NavBar.css';
+import '../styles/MealList.css';
+
+const NavBar = () => (
+  <nav data-testid="nav" className="nav">
+    <Link className="back" to="/" onClick={() => MealList()}>
```

Comment:
- [ ] Good job on implementing some of the suggested changes by the previous reviewer 👏 However, as suggested, the back button should not lead to the reloading of the page. Basically, the way it is supposed to be working is to be able to go back to the previous URL as stored in the browser history stack. There are different ways that can be used to implement this. Either by using `useNavigate` hook or `useHistory` hook from react-router dom. You can try both ways and then see what's working better 😉  Below I have provided some hint by using useHistory. Maybe can help you see the implementation but make sure you try the useNavigate also 👇 

```js
import { useHistory } from "react-router-dom";

export const Item = () => {
    let history = useHistory();
    return (
        <>
          <button onClick={() => history.goBack()}>Back</button>
        </>
    );
};
```

Reply:
@BertrandConxy Thank you

Reply:
Hi @BertrandConxy 
I have gone through the project requirements [here](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=the%20details%20page.-,Details%20page,-In%20the%20details)
Upon clicking back ` '<' (Back)`, the user should be taken to the `home page` and not the previous `page` as you had suggested, kindly confirm this from project requirements [here](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=the%20details%20page.-,Details%20page,-In%20the%20details)
Thank you.